src/ds3/ds3.go

```diff
@@ -1,20 +1,20 @@
 package ds3
 
 import (
-    "ds3/net"
+    "ds3/networking"
     "net/url"
 )
 
 type Client struct {
-    netLayer net.Network
+    netLayer networking.Network
 }
 
 type Builder struct {
-    connectionInfo *net.ConnectionInfo
+    connectionInfo *networking.ConnectionInfo
 }
 
-func NewBuilder(endpoint *url.URL, creds net.Credentials) *Builder {
-    return &Builder{&net.ConnectionInfo{
+func NewBuilder(endpoint *url.URL, creds networking.Credentials) *Builder {
```

Comment:
Can we call the combination of NewBuilder & Builder something that better describes what's being built?